### ISSUE TITLE
Hi, my keyboard is not being recoginized by uefi/bios or grub

### ISSUE THREAD

**Issue by sionka5** [KMKfw/kmk_firmware#898]
Hi, Im not sure if its bug but my rp2040 based keyboard doesnt want to show up in my uefi or grub bootloader. I dont know what to do, maybe someone have fix for this?

**Comment by xs5871** [comment]
Yes: [discussion in chat](https://kmkfw.zulipchat.com/#narrow/stream/358347general/topic/No.20UEFI.20with.20KMK.2C.20RP2040.2C.20CPython.208.2E2.2E6).

**Comment by Abn0rm4l** [comment]
Link appears broken.

**Comment by xs5871** [comment]
This is the[correc link.](https://kmkfw.zulipchat.com/#narrow/stream/358347-general/topic/No.20UEFI.20with.20KMK.2C.20RP2040.2C.20CPython.208.2E2.2E6)
Anyway, I'm closing this issue because it's not a bug. Help requests are handled in chat.
